helm-service/chart/templates/deployment.yaml

```diff
@@ -30,11 +30,7 @@ spec:
         - name: helm-service
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          {{- if .Values.image }}
-          image: {{ .Values.image }} # use image from .Values.image (e.g., when starting via skaffold)
-          {{- else }}
-          image: "{{ .Values.helmservice.image.repository }}:{{ .Values.helmservice.image.tag | default .Chart.AppVersion }}"
-          {{ end }}
+          image: {{ include "service.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.AppVersion) }}
```

Comment:
would it make sense to create a template function for `{{ include "service.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.AppVersion) }}`, given that this line is also used in the `helm-service.labels` function in `_helpers.tpl`?

Reply:
Just saw that those lines are not completely the same, so I believe my comment above is obsolete